### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by Bodemola123** [rainbow-me/rainbowkit#2173] [Feedback]
Sorry how can i solve this issue. I have done the installation down to the dot as it is in the website but it brings this error "Error: (0 , _rainbow_me_rainbowkit__WEBPACK_IMPORTED_MODULE_0__.getDefaultConfig) is not a function"

**Comment by DanielSinclair** [comment]
@Bodemola123 This seems like a bundler issue. Can you open a GitHub Issue with an example repo/project?
